### PR TITLE
Changed TimestampFormat for server logs

### PR DESCRIPTION
**Pull request by Baxxster**
Thanks for implementing the function in the first place :)

I'd like to change it to this pattern, because I got really confused when I opened my dedicated server window and all I saw was different numbers, there wasn't a lot of activity on my servers that day, so first I thought this was game timer or something, but with this it's easier to see exactly when things happen. This is also good for people who like to make scripts with "grep" and create stats out of that :)

### CODE REVIEW

File: OpenRA.Game/Settings.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public class ServerSettings
 		[Desc("Disallow AI bots.")]
 		public bool LockBots = false;
 
-		public string TimestampFormat = "HH:mm";
+		public string TimestampFormat = "dd/MM - HH:mm:ss";
```
Comment (pchote) [inline OpenRA.Game/Settings.cs]: Could we use [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) (`YYYY-MM-ddTHH:mm:ssZ`, but there's probably a short code for it) instead?

Comment (ScottNZ) [inline OpenRA.Game/Settings.cs]: https://msdn.microsoft.com/en-us/library/az4se3k1%28v=vs.110%29.aspx#Roundtrip
Looks like it's "o"

Comment (baxxxster) [inline OpenRA.Game/Settings.cs]: Yes, I was afraid of suggesting too long string for prettiness-sake, "s" seems to be: YYYY-MM-ddTHH:mm:ss (No Z at the end, but could probably be added if needed)

Comment (baxxxster) [inline OpenRA.Game/Settings.cs]: "o" seems to also include the microseconds which might be "too much" again, there shouldn't be many events on the server that requires deeper logging than seconds afaik?

**Comment by baxxxster** [comment]
Updated the format as requested

**Comment by RoosterDragon** [comment]
's' seems very reasonable.

:+1:

**Comment by Mailaender** [comment]
Indeed. :+1: It is not seconds like I first thought. https://msdn.microsoft.com/en-us/library/az4se3k1%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396#Sortable

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/f3a6b6a6b023fa3622fef222664b315a41ae0ad0...002d75c202aa99a3d3643ed24b463b8fb3153e35)

**Pull request merged**